app/src/lib/editors/win32.ts

```diff
@@ -172,9 +172,17 @@ const editors: IWindowsExternalEditor[] = [
       publisher === 'Microsoft Corporation',
   },
   {
-    name: 'Sublime Text',
+    name: 'Sublime Text 3',
     registryKeys: [LocalMachineUninstallKey('Sublime Text 3_is1')],
     executableShimPath: ['subl.exe'],
+    expectedInstallationChecker: (displayName, publisher) =>
+      displayName.startsWith('Sublime Text 3') &&
+      publisher === 'Sublime HQ Pty Ltd',
+  },
+  {
+    name: 'Sublime Text',
+    registryKeys: [LocalMachineUninstallKey('Sublime Text_is1')],
+    executableShimPath: ['subl.exe'],
     expectedInstallationChecker: (displayName, publisher) =>
       displayName.startsWith('Sublime Text') &&
       publisher === 'Sublime HQ Pty Ltd',
```

Comment:
For laughs and completeness can we also add Sublime Text 2 support, I would also suggest ordering from newest to oldest?

```suggestion
    name: 'Sublime Text',
    registryKeys: [LocalMachineUninstallKey('Sublime Text_is1')],
    executableShimPath: ['subl.exe'],
    expectedInstallationChecker: (displayName, publisher) =>
      displayName === 'Sublime Text' &&
      publisher === 'Sublime HQ Pty Ltd',
},
{
    name: 'Sublime Text 3',
    registryKeys: [LocalMachineUninstallKey('Sublime Text 3_is1')],
    executableShimPath: ['subl.exe'],
    expectedInstallationChecker: (displayName, publisher) =>
      displayName === 'Sublime Text 3' &&
      publisher === 'Sublime HQ Pty Ltd',
},
{
    name: 'Sublime Text 2',
    registryKeys: [LocalMachineUninstallKey('Sublime Text 2_is1')],
    executableShimPath: ['sublime_text.exe'],
    // Sublime Text 2 doesn't define a publisher
    expectedInstallationChecker: (displayName, _) =>
      displayName.startsWith('Sublime Text 2'),
```